administrator/components/com_joomgallery/helpers/adminconfig.php

```diff
@@ -113,6 +116,8 @@ public function save($newconfig = null, $id = false, $existing_row = 0, $group_i
       return false;
     }
 
+    $this->_mainframe->triggerEvent('onContentAfterSave', array(_JOOM_OPTION.'.config', &$config, $isNew));
```

Comment:
$this->_mainframe is not declared in this content, this call will throw an error

Reply:
solved